pkg/credentialprovider/azure_credentials_test.go

```diff
@@ -205,8 +205,8 @@ func TestProcessImageWithMirrorMapping(t *testing.T) {
 	for _, test := range testcases {
 		t.Run(test.description, func(t *testing.T) {
 			targetloginServer, sourceloginServer := acrProvider.parseACRLoginServerFromImage(test.image)
-			assert.Equal(t, targetloginServer, test.expectedLoginServer)
```

Comment:
For assert.Equal, expected variable comes before actual variable.
https://github.com/stretchr/testify/blob/master/assert/assertions.go#L460